mathics/core/pattern.py

```diff
@@ -2,11 +2,12 @@
 # cython: profile=False
 # -*- coding: utf-8 -*-
 
-
+from mathics.core.symbols import Symbol
 from mathics.core.expression import Expression
 from mathics.core.symbols import system_symbols, ensure_context, Atom, Symbol
 from mathics.core.util import subsets, subranges, permutations
 from itertools import chain
+from mathics.core.systemsymbols import SymbolFlat, SymbolOrderless, SymbolOneIdentity
```

Comment:
Neither this.

Reply:
Now used anymore because now we use the bitmask